README.md

```diff
@@ -1 +1,29 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+Ultra safe and nice website with some geek jokes.
+
+Requirements:
+    Docker desktop,
+    Postgresql
+
+To run it by your own you shold make an docker container using:
+
+docker run  -d \                     
```

Comment:
you still did not fix this, i cant even run docker container for your project :(